Index: dist/src/service/post-service.d.ts
===================================================================
diff --git a/dist/src/service/post-service.d.ts b/dist/src/service/post-service.d.ts
deleted file mode 100644
--- a/dist/src/service/post-service.d.ts	
+++ /dev/null	
@@ -1,6 +0,0 @@
-export declare class PostService {
-    private postRepo;
-    private randomId;
-    constructor();
-    create: (data: any) => Promise<void>;
-}
Index: dist/src/service/post-service.js.map
===================================================================
diff --git a/dist/src/service/post-service.js.map b/dist/src/service/post-service.js.map
deleted file mode 100644
--- a/dist/src/service/post-service.js.map	
+++ /dev/null	
@@ -1,1 +0,0 @@
-{"version":3,"file":"post-service.js","sourceRoot":"","sources":["../../../src/service/post-service.ts"],"names":[],"mappings":";;;AAAA,+CAA0C;AAC1C,2CAAqC;AAErC,MAAa,WAAW;IAIpB;QAIA,WAAM,GAAG,KAAK,EAAE,IAAS,EAAC,EAAE;QAC5B,CAAC,CAAA;QAJG,IAAI,CAAC,QAAQ,GAAG,IAAI,mBAAQ,EAAE,CAAA;QAC9B,IAAI,CAAC,QAAQ,GAAG,IAAI,oBAAQ,EAAE,CAAA;IAClC,CAAC;CAGJ;AAVD,kCAUC"}
\ No newline at end of file
Index: dist/src/controller/post-controller.js.map
===================================================================
diff --git a/dist/src/controller/post-controller.js.map b/dist/src/controller/post-controller.js.map
deleted file mode 100644
--- a/dist/src/controller/post-controller.js.map	
+++ /dev/null	
@@ -1,1 +0,0 @@
-{"version":3,"file":"post-controller.js","sourceRoot":"","sources":["../../../src/controller/post-controller.ts"],"names":[],"mappings":""}
\ No newline at end of file
Index: dist/src/router/router-post.js.map
===================================================================
diff --git a/dist/src/router/router-post.js.map b/dist/src/router/router-post.js.map
deleted file mode 100644
--- a/dist/src/router/router-post.js.map	
+++ /dev/null	
@@ -1,1 +0,0 @@
-{"version":3,"file":"router-post.js","sourceRoot":"","sources":["../../../src/router/router-post.ts"],"names":[],"mappings":""}
\ No newline at end of file
Index: dist/src/service/post-service.js
===================================================================
diff --git a/dist/src/service/post-service.js b/dist/src/service/post-service.js
deleted file mode 100644
--- a/dist/src/service/post-service.js	
+++ /dev/null	
@@ -1,15 +0,0 @@
-"use strict";
-Object.defineProperty(exports, "__esModule", { value: true });
-exports.PostService = void 0;
-const postRepo_1 = require("../repo/postRepo");
-const random_id_1 = require("./random-id");
-class PostService {
-    constructor() {
-        this.create = async (data) => {
-        };
-        this.postRepo = new postRepo_1.PostRepo();
-        this.randomId = new random_id_1.RandomId();
-    }
-}
-exports.PostService = PostService;
-//# sourceMappingURL=post-service.js.map
\ No newline at end of file
Index: dist/src/router/router-post.js
===================================================================
diff --git a/dist/src/router/router-post.js b/dist/src/router/router-post.js
deleted file mode 100644
--- a/dist/src/router/router-post.js	
+++ /dev/null	
@@ -1,1 +0,0 @@
-//# sourceMappingURL=router-post.js.map
\ No newline at end of file
Index: dist/src/controller/post-controller.js
===================================================================
diff --git a/dist/src/controller/post-controller.js b/dist/src/controller/post-controller.js
deleted file mode 100644
--- a/dist/src/controller/post-controller.js	
+++ /dev/null	
@@ -1,1 +0,0 @@
-//# sourceMappingURL=post-controller.js.map
\ No newline at end of file
diff --git a/dist/src/controller/post-controller.d.ts b/dist/src/controller/post-controller.d.ts
deleted file mode 100644
diff --git a/dist/src/router/router-post.d.ts b/dist/src/router/router-post.d.ts
deleted file mode 100644
